honeybee_grasshopper_energy/src/HB Weekly Schedule.py

```diff
@@ -0,0 +1,147 @@
+            whole day.
+        _wed: A list of 24 values that represent the schedule values at each
+            hour of Wednesday. This can also be a single constant value for the
+            whole day.
+        _thu: A list of 24 values that represent the schedule values at each
+            hour of Thursday. This can also be a single constant value for the
+            whole day.
+        _fri: A list of 24 values that represent the schedule values at each
+            hour of Friday. This can also be a single constant value for the
+            whole day.
+        _sat: A list of 24 values that represent the schedule values at each
+            hour of Saturday. This can also be a single constant value for the
+            whole day.
+        _holiday_: An optional list of 24 values that represent the schedule
+            values at each hour of holidays. This can also be a single constant
+            value for the whole day. If no values are input here, the schedule
+            for Sunday will be used for all holidays.
+        _summer_des_: An optional list of 24 values that represent the schedule
+            values at each hour of the summer design day. This can also be a
+            single constant value for the whole day. If None, the daily
+            schedule with the highest average value will be used.
+        _winter_des_: An optional list of 24 values that represent the schedule
+            values at each hour of the summer design day. This can also be a
+            single constant value for the whole day. If None, the daily
+            schedule with the lowest average value will be used.
+        _name: A text string representing a name for the schedule.  This name
+            should be unique among the schedules in your Grasshopper document
+            to ensure that there are no naming conflicts in the resulting
+            simulation files.
+        _type_limit_: A text string from the name of the ScheduleTypeLimit to
+            be looked up in the schedule type limit library. This can also be a
+            custom ScheduleTypeLimit object from the "HB Type Limit" component.
+            The input here will be used to validate schedule values against
+            upper/lower limits and assign units to the schedule values. Default:
+            "Fractional" for values that range continuously between 0 and 1.
+            Choose from the following built-in options:
+                * Fractional
+                * On-Off
+                * Temperature
+                * Activity Level
+                * Power
+                * Humidity
+                * Angle
+                * Delta Temperature
+    
+    Returns:
+        report: Reports, errors, warnings, etc.
+        ----------: ...
+        schedule: A ScheduleRuleset object that can be assigned to a Room, a Load
+            object, or a ProgramType object.
+        ----------: ...
```

Comment:
Can we remove these separators?

Reply:
It definitely makes the component easier to read and it doesn't increase the size of the component at all. Why do you want to remove them?  And do you mean just remove the docstring here or remove them on the actual ghuser object?

Reply:
I mean removing it from both docstring and the component itself. It was a solution for really long components and now that we are doing a better job in keeping the components small enough we should not be needing them anymore.

Reply:
Ok. I'll remove it.

Reply:
Done.